orca-clouddriver/src/main/java/com/netflix/spinnaker/orca/clouddriver/tasks/cache/ClouddriverClearAltTablespaceTask.java

```diff
@@ -48,18 +47,21 @@ public TaskResult execute(@Nonnull StageExecution stage) {
           result.getOrDefault("tables", emptyList()));
 
       return TaskResult.builder(ExecutionStatus.SUCCEEDED).context(result).build();
-    } catch (RetrofitError e) {
-      Map<String, Object> output = new HashMap<>();
-      List<String> errors = new ArrayList<>();
+    } catch (SpinnakerHttpException e) {
+      output = new HashMap<>();
 
-      if (e.getResponse() != null && e.getResponse().getBody() != null) {
-        String error = new String(((TypedByteArray) e.getResponse().getBody()).getBytes());
-        log.error("Failed clearing clouddriver table namespace: {}", error, e);
-        errors.add(error);
+      if (e.getResponseBody() != null) {
+        Map<String, Object> errorBody = e.getResponseBody();
```

Comment:
nit: assign this above the if to only call `e.getResponseBody()` once.

Reply:
This suggestion is no more relevant as the response body will no more be considered wrt the latest commit